src/Core/AdminConsole/Services/Implementations/EventIntegrations/IntegrationTemplatePropertyCache.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Concurrent;
```

Comment:
🎨 All this is something that I think is so generic and reusable that it could be in a more central / core location, for usage beyond event integrations.